front/src/Phaser/Game/GameScene.ts

```diff
@@ -745,6 +759,23 @@ ${escapedMessage}
                 },
             });
         });
+
+        iframeListener.disablePlayerControlStream.subscribe(()=>{
+            this.userInputManager.clearAllKeys();
+        })
+        iframeListener.enablePlayerControl.subscribe(()=>{
+            this.userInputManager.restoreAllKeys();
+        })
+        let sprite : Sprite;
```

Comment:
We should rename "sprite" to something more obvious.
Like "scriptedBubbleSprite"